documents/utils.py

```diff
@@ -17,6 +22,29 @@ def get_document_attachment_directory_path(instance):
     return f"{settings.ATTACHMENT_MEDIA_DIR}/{instance.id}/"
 
 
+def get_decrypted_file(file, file_name):
+    nonce = file[:16]
+    # Perform same nonce checks here as Pycryptodome, so we can raise a more user
+    # friendly error message
+    if not isinstance(nonce, (bytes, bytearray, memoryview)) or len(nonce) != 16:
+        raise ValueError("Data is corrupted.")
+    tag = file[16:32]
+    cypher_text = file[32:]
+    counter = 0
+    num_keys = len(FIELD_ENCRYPTION_KEYS)
+    while counter < num_keys:
```

Comment:
I suppose this could be for key in FIELD_ENCRYPTION_KEYS, but this works as well :)